client-src/elements/utils.js

```diff
@@ -62,3 +62,42 @@ export function findFirstFeatureStage(intentStage, currentStage, fe) {
 export function flattenSections(stage) {
   return stage.sections.reduce((combined, section) => [...combined, ...section.fields], []);
 }
+
+/* Set up scrolling to a hash url (e.g. #id_explainer_links). */
+export function setupScrollToHash(pageElement) {
+  const scrollToElement = (hash) => {
+    if (hash) {
+      const el = pageElement.shadowRoot.querySelector(hash);
+      if (el) {
+        // Find the form field element.
+        const fieldRowSelector = `chromedash-form-field[name="${el.name}"] tr + tr`;
```

Comment:
Also, what if we wanted to visually highlight the target form field?  Could we add a CSS class or something?

Reply:
The focus() call on the form field itself highlights the target form field better than we could do by yet another highlight style.  

I'll change the names as you suggest since this is specific to the feature form fields, laid out in the table structure. 
But I was thinking this might be generalizable (later) to pass in the selectors, or pass in a page-specific function that would generate the appropriate selector.